server_cmds.go

```diff
@@ -368,7 +368,7 @@ func (s *ServerMethods) PrivilegeKeyAdd(ttype, id1, id2 int, options ...CmdArg)
 		Token string
 	}{}
 	options = append(options, NewArg("tokentype", ttype), NewArg("tokenid1", id1), NewArg("tokenid2", id2))
-	_, err := s.ExecCmd(NewCmd("privilegekeylist").WithArgs(options...).WithResponse(&t))
+	_, err := s.ExecCmd(NewCmd("privilegekeyadd").WithArgs(options...).WithResponse(&t))
```

Comment:
This looks like the only needed change

Reply:
I forgot to close the pull request. Since your repository hasn’t been updated for a long time, I forked it and made the changes myself.

Reply:
> This looks like the only needed change

I forgot that the pull request here is based on the main branch after I forked it. Sorry about that.

Reply:
> This looks like the only needed change

I have opened a new pull request, please review it.